2.ServiceTracker_App/Components/tracker-data-component.yaml

```diff
@@ -69,31 +68,30 @@ spec:
             successThreshold: 1
             timeoutSeconds: 5 
   parameters:
-    - name: dbuser
-      description: database username
-      type: string
-      required: false
-    - name: dbpassword
-      description: database password
-      type: string
+    - name: dbsecret
+      description: secret with database connection information
       required: false
+      fieldPaths:
+      - spec.containers[0].env[0].fromSecret.name
```

Comment:
What does it mean to have three fieldPaths?

Reply:
It means that the value passed for this parameter will populate each of the fields in the array of field paths :+1:

Reply:
I am not a fan of this solution to pass in param values but that is what the v1alpha2 spec says.